engine/flink/generic/src/main/resources/defaultModelConfig.conf

```diff
@@ -7,19 +7,24 @@
 
   timeout: 10s
   checkpointConfig {
-      checkpointInterval: 10m
+    checkpointInterval: 10m
   }
   rocksDB {
     enable: true
   }
-
-  kafka {
-    kafkaAddress: ${?KAFKA_ADDRESS}
-    kafkaProperties {
-      "schema.registry.url": ${?SCHEMA_REGISTRY_URL}
+  components {
+    kafka {
+      config {
+        kafka {
```

Comment:
do we need so much nesting?

Reply:
right now its nested, but we discused it with @arkadius that config should already be kafka config

Reply:
I've flattened the structure